### PR TITLE
feat: add absolute uri to djrf-api audit decorator

### PR DESCRIPTION
**Pull request by johanseto**
(cherry picked from commit 9a0dc8979f23e9835c613a1cead30cad0b59f291)

## Description

This PR is like a backport or something like that but to a branch we can manage in Nelp.
https://github.com/eduNEXT/eox-audit-model/pull/25

## Testing instructions

Please provide detailed step-by-step instructions for testing this change.

## Additional information

Include anything else that will help reviewers and consumers understand the change.
- Does this change depend on other changes elsewhere?
- Any special concerns or limitations? For example: deprecations, migrations, security, or accessibility.
- Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.

## Checklist for Merge

- [ ] Tested in a remote environment
- [ ] Updated documentation
- [ ] Rebased master/main
- [ ] Squashed commits

### CODE REVIEW

**Comment by andrey-canon** [comment]
I would like to know the reason, why do you need that? why aren't you copying the eox-tagging approach ?

**Comment by johanv26** [comment]
> I would like to know the reason, why do you need that? why aren't you copying the eox-tagging approach ?

Because If eox-tagging has djrf-decorator we would use it for rest views not?

**Comment by andrey-canon** [comment]
Ok, now I understand your point, I think the eox-tagging approach is still valid  and in terms of code could be almost the  same and I really think this is a good improvement however this is a custom extra change  that we don't want to maintain so first I prefer to know the opinion of  @santiagosuarezedunext,  could you talk to him ? if he consider that this could be included in the future we can continue with this otherwise we should close it and use the eox-tagging approach

File: eox_audit_model/decorators.py [hunk 1]
```diff
@@ -96,11 +96,11 @@ def wrapper(*args, **kwargs):  # pylint: disable=missing-docstring
 
             @audit_method(action=action)
             @rename_function(name=method_name)
-            def api_method(*input_parameters):  # pylint: disable=unused-argument
+            def api_method(*input_parameters, absolute_uri=""):  # pylint: disable=unused-argument
                 """This method is just a wrapper in order to capture the input data"""
                 return func(*args, **kwargs)
 
-            return api_method(*input_parameters)
+            return api_method(*input_parameters, absolute_uri=request.build_absolute_uri())
```
Comment (andrey-canon) [inline eox_audit_model/decorators.py:103]: could you consider something like 
```suggestion
            return api_method(*input_parameters, **request_parameters)
```

where `request_parameters` is something like 

```
"request_paramneters": {
  "absolute_uri":request.build_absolute_uri(),
  ... more things that could be considered important 
}
```

**Comment by Asespinel** [comment]
@santiagosuarezedunext Do you consider this change is necesarry or we should use eox tagging instead?

**Comment by MaferMazu** [comment]
This change was merged in #25. Do we need this PR to `mango.nelp`, or can we close it? @johanseto

**Comment by johanseto** [comment]
> This change was merged in #25. Do we need this PR to `mango.nelp`, or can we close it? @johanseto

Thanks, @MaferMazu for the support, I am going to close it.

**Pull request closed without merging**